simply-rets-admin.php

```diff
@@ -49,8 +49,26 @@ public static function register_admin_settings() {
       register_setting('sr_admin_settings', 'sr_agent_office_above_the_fold');
       register_setting('sr_admin_settings', 'sr_show_mls_trademark_symbol');
       register_setting('sr_admin_settings', 'sr_disable_listing_details_map');
+      register_setting('sr_admin_settings', 'sr_date_default_timezone', array(
+          "default" => ""
+      ));
   }
 
+  public static $timezones = array(
```

Comment:
I'm not sure if this is actually necessary, but it can be set to "nothing" and will at least provide an extra layer of options for people who have issues with timestamps.